src/common/dao/repository.go

```diff
@@ -114,12 +126,16 @@ func GetTotalOfRepositories(query ...*models.RepositoryQuery) (int64, error) {
 }
 
 // GetRepositories ...
-func GetRepositories(query ...*models.RepositoryQuery) ([]*models.RepoRecord, error) {
+func GetRepositories(sorting string, query ...*models.RepositoryQuery) ([]*models.RepoRecord, error) {
```

Comment:
How about putting the `sorting` into the `query` as a field just like the `Pagination` in https://github.com/goharbor/harbor/blob/master/src/common/models/repo.go#L42? Maybe `sorting` can be a struct so that other APIs can reuse it.

Reply:
OK